ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -97,7 +97,7 @@ type ssh_daemon_options_type = {
     "AcceptEnv" ? string[]
     "AllowAgentForwarding" ? legacy_binary_affirmation_string
     "AllowGroups" ? string
-    "AllowTcpForwarding" ? legacy_binary_affirmation_string
+    "AllowTcpForwarding" ? string with match (SELF, '^yes|no|all|local|remote$')
```

Comment:
choice('yes', 'no', 'all', 'local', 'remote')

is probably better here than a regex